frontend/src/components/Login.js

```diff
@@ -1,51 +1,47 @@
 import { Form, Button } from 'react-bootstrap'
 import { useState } from 'react'
+import { loginUser } from '../util/requests'
+import { ERR_INVALID_INPUT } from '../util/constants'
+import styles from '../styles/components/auth.module.css';
 
-const Login = () => {
-
+const Login = ({ setLoggedIn }) => {
     const [error, setError] = useState(false)
     const [data, setData] = useState({})
-    const [user, setUser] = useState({})
 
     const handleSubmit = async e => {
         e.preventDefault()
-        console.log(data)
 
         if(!data.email || !data.password) {
-            setError(true)
+            setError(ERR_INVALID_INPUT)
             return
         }
 
-        const fetchedUser = await loginUser(data)
-        console.log(fetchedUser)
-    }
+        const res = await loginUser(data)
+
+        if(res.errMsg)
+            setError(res.errMsg)
+        else {
+            setError(false)
+            setLoggedIn(true)
+            localStorage.setItem("tokens", JSON.stringify(res))
+        }
 
-    const loginUser = async data => {
-        const fetchedUser = await fetch("http://localhost:4000/api/login", {
-            method: 'post',
-            headers: {
-                'Content-type': 'application/json',
-                'Accept': 'application/json'
-            },
-            body: JSON.stringify(data),
-        })
-
-        return fetchedUser.json()
+        console.log(res);
```

Comment:
**Nicht relevant:**
`console.log()` in production code?

Reply:
Erledigt.